src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -358,6 +358,7 @@ public Description getDescription() {
     public void run(final RunNotifier notifier) {
         EachTestNotifier testNotifier = new EachTestNotifier(notifier,
                 getDescription());
+        testNotifier.fireTestSuiteStarted();
```

Comment:
It might be easier to determine if this is a problem (and if so, the best way to address it) if you could describe the problem you are trying to solve with this change and/or how you plan to use the API